src/App.tsx

```diff
@@ -1,35 +1,29 @@
 import { useState } from 'react'
-import reactLogo from './assets/react.svg'
-import viteLogo from '/vite.svg'
-import './App.css'
+import Counter from './components/counter';
+import Header from './components/header/header';
+import './App.css';
 
 function App() {
-  const [count, setCount] = useState(0)
+  let genreList=["Horror","Romantic","Thrilling", "Comedy"];
+  const [selectedGenre, setSelectedGenre] = useState('Horror');
 
+  const onSelectGenre = (genre:string) => () => {
```

Comment:
Genre is not a simple string - it is one of the elements of genre list. Use typescript functionaslity to protect this value.